superset/datasets/commands/create.py

```diff
@@ -47,9 +46,7 @@ def run(self) -> Model:
             # Creates SqlaTable (Dataset)
             dataset = DatasetDAO.create(self._properties, commit=False)
 
-            # Updates columns and metrics from the dataset
-            dataset.metrics = [SqlMetric(metric_name="COUNT(*)", expression="COUNT(*)")]
```

Comment:
@hughhhh would you mind linking to where this `COUNT(*)` metric is being created?

Reply:
`dataset.fetch_metadata` calls `database.get_metrics`

https://github.com/preset-io/superset/blob/e42625c8fdd26162f232d1b2489a2326d8116771/superset/db_engine_specs/base.py#L1248